source/monitoring/metrics-alerts/collect-minio-metrics-using-prometheus.rst

```diff
@@ -0,0 +1,177 @@
+.. _minio-metrics-collect-using-prometheus:
+
+======================================
+Collect MinIO Metrics Using Prometheus
+======================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+MinIO leverages `Prometheus <https://prometheus.io/>`__ for metrics and alerts.
+MinIO publishes Prometheus-compatible scraping endpoints for cluster and
+node-level metrics. See :ref:`minio-metrics-and-alerts-endpoints` for more
+information.
+
+The procedure on this page documents scraping the MinIO metrics
+endpoints using a Prometheus instance, including deploying and configuring
+a simple Prometheus server for collecting metrics. 
+
+This procedure is not a replacement for the official
+:prometheus-docs:`Prometheus Documentation`. Any specific guidance
+related to configuring, deploying, and using Prometheus is made on a best-effort
+basis.
+
+Requirements
+------------
+
+Install and Configure ``mc`` with Access to the MinIO Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This procedure uses :mc:`mc` for performing operations on the MinIO
+deployment. Install :mc:`mc` on a machine with network access to the
+deployment. See the ``mc`` :ref:`Installation Quickstart <mc-install>` for
```

Comment:
My preference is probably for anything that provides a link to the element since you don't know where a reader is coming from or if they have any familiarity with a given concept.  But in this case becomes a bit visually distracting having a bunch of red mc's everywhere.  If I had to choose I would go with linking.  In either case this change doesn't need to block the PR.